backend/core/vite.config.ts

```diff
@@ -0,0 +1,9 @@
+/// <reference types="vitest" />
```

Comment:
do you really need this ? I just switch to vitest on one of my project. I didn't need this.